app/services/service_provider_config.rb

```diff
@@ -0,0 +1,22 @@
+class ServiceProviderConfig
+  def initialize(filename:, issuer:)
```

Comment:
Why are you using this syntax here in particular?  Should we start using it everywhere?  What are the rules for when we should or shouldn't use this?